### PR TITLE
Update msal-browser README

### PR DESCRIPTION
**Pull request by jo-arroyo**
This PR updates the msal-browser README.md in the following ways:
- Add CDN for installation
- Update sample links in the msal-browser README.md
- Add links to to the TypeScriptTestApp2.0 sample
- Correct typos

This PR addresses #1976

### CODE REVIEW

File: lib/msal-browser/README.md [hunk 1]
```diff
@@ -86,18 +86,19 @@ If you have MSAL v1.x currently running in your application, you can follow the
 
 ## Samples
 
-The [`VanillaJSTestApp2.0` folder](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples) contains sample applications for our libraries. You can run any sample by changing the `authConfig.js` file in the respective folder to match your app registration and running the `npm` command `npm start -- -s <sample-name> -p <port>`.
+The [`msal-browser-samples` folder](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-browser-samples) contains sample applications for our libraries. You can run any VanillaJSTestApp2.0 sample by changing the `authConfig.js` file in the respective folder to match your app registration and running the `npm` command `npm start -- -s <sample-name> -p <port>` in the VanillaJSTestApp2.0 folder. The TypescriptTestApp2.0 sample can be run with `npm start` in its folder.
```
Comment (sameerag) [inline lib/msal-browser/README.md:89]: Does it makes sense to itemize the points mentioned here regarding samples. Minor nitpick though.

Comment (jo-arroyo) [reply lib/msal-browser/README.md]: Done.

**Comment by jasonnutter** [comment]
@jhutchings1 Any insight into why the CodeQL step is failing? Thanks!

**Comment by jhutchings1** [comment]
@jasonnutter Error #32 means that there was no source seen in a particular language. Looks like an issue with the Java analysis. The quick solution is to disable Java and focus on the JS instead. I'll send you a PR.

**Pull request merged**